configure.ac

```
@@ -2419,6 +2419,33 @@ AC_LINK_IFELSE(
     AC_DEFINE([HAS_BSD_GETAFFINITY_NP], [1])],
     [AC_MSG_RESULT([pthread_getaffinity_np not found])])])
 
+## prctl
+AC_CHECK_DECL(
+  [prctl],
+  [AC_DEFINE([HAS_DECL_PRCTL] , [1])],
+  [],
+  [#include <sys/prctl.h>])
+
+## pthread_setname_np
+AC_CHECK_FUNCS(
+  [pthread_setname_np],
+  [AC_DEFINE([HAS_PTHREAD_SETNAME_NP], [1])])
+
+## pthread_set_name_np
+AC_CHECK_FUNCS(
+  [pthread_set_name_np],
+  [AC_DEFINE([HAS_PTHREAD_SET_NAME_NP], [1])])
+
+## SetThreadDescription
+AC_CHECK_FUNCS(
+  [SetThreadDescription],
+  [AC_DEFINE([HAS_SETTHREADDESCRIPTION], [1])])
+
+AC_CHECK_DECL(
+  [SetThreadDescription],
+  [AC_DEFINE([HAS_DECL_SETTHREADDESCRIPTION], [1])],
+  [#include <processthreadsapi.h>])
```

```suggestion
  [AC_DEFINE([HAS_DECL_SETTHREADDESCRIPTION], [1])], []
  [#include <processthreadsapi.h>])
```

The include was mistakenly placed in the _action-if-not-found_ block. See [`AC_CHECK_DECL`](https://www.gnu.org/software/autoconf//manual/autoconf-2.71/html_node/Generic-Declarations.html#index-AC_005fCHECK_005fDECL-1).